packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -11,6 +11,10 @@ import {
 /**
  * @nativeHandler onClick
  */
+
+/**
+ * @nativeHandler onKeyPress
```

Comment:
@andresriveratoro I needed to expose this native event in `va-button` so that the file upload button (which uses `va-button`) could be activated by keyboard for a11y. Are there any issues with this that you can think of or anything else I need to consider by adding this here?